app/2.1.x/clustering.md

```diff
@@ -90,7 +90,7 @@ greatly reduces the load on the main database instance since read-only
 queries are no longer sent to it.
 
 To learn more about how to configure this feature, refer to the
-<a href="/{{page.kong_version}}/configuration/#datastore-section">Datastore section</a>
+[Datastore section](/{{page.kong_version}}/property-reference/#datastore-section)
```

Comment:
```suggestion
[Datastore section](/{{page.kong_version}}/configuration/#datastore-section)
```
Looks like there was some confusion about the link/page name here. It's `configuration` for Community docs, and `property-reference` for Enterprise docs. Couldn't tell you why they're different...